projects/plugins/jetpack/functions.opengraph.php

```diff
@@ -11,7 +11,7 @@
  * @package automattic/jetpack
  */
 
-add_action( 'wp_head', 'jetpack_og_tags' );
+add_action( 'wp_head', 'jetpack_og_tags', 100 );
```

Comment:
This is definitely the correct way to output the OG tags later than most plugins, but looking at it like this it does worry me a bit about unintended consequences unrelated to SIG. I wonder if there's a way we could add a filter (something like `jetpack_og_tags_priority`) and set it to 100 only when SIG is active. What do you think?

cc @pablinos for any thoughts as well

Reply:
Thinking about this, people might be removing this action or assuming it's priority is the default.

Maybe rather than changing this action we have our own for the SIG OG tags? Then we won't break compatibility.